codefresh/pipeline-creator/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Create Codefresh pipelines'
+description: 'GitHub Action that creates Codefresh pipelines from templated spec files'
+author: 'Cloud Posse <hello@cloudposse.com>'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+branding:
+  icon: 'activity'
+  color: 'green'
```

Comment:
Define input parameters.

Reply:
For example: https://github.com/cloudposse/actions/blob/master/github/slash-command-dispatch/action.yml#L3-L22